evennia/contrib/actions/README.md

```diff
@@ -0,0 +1,372 @@
+
+at_failure (function) - to be called when the action has been attempted and
+                        has failed. Takes the action itself as its argument.
+
+at_cancel (function) - to be called when the action has been cancelled. Takes
+                       the action itself as its argument.
+
+You must supply at least a key, a desc, an owner and a room. The key and desc can sit in the class definition, but the owner and the room must be unique to each instance of your class. Some desc examples include "performing an action", "eating $t" and "giving $d to $t". They should generally be of this form, i.e. with a verb in "ing" form somewhere at the start of the desc.
+
+Each action employs bodyparts, and the same character cannot perform two given actions at once when they use the same bodyparts. These can include actual bodyparts such as "torso" and "left arm", as well as more abstract concepts, such as "movement action" or "combat action" that are meant to group actions together. Each bodypart is simply a string, and can be cross-referenced with any body part objects or containers that might be attached to your character. Beyond serving as a means to keep certain actions from happening at once, you can associate them with movement types.
+
+If you intend to eschew using bodyparts altogether, you should still supply actions with a single bodypart, possibly called "body" or "action" (the name will show up in a few messages to the characters, so do not call it something silly), because when an action has no bodyparts, any number of copies of it can be initiated by the same character at a given time. You do not want this to happen, especially when it comes to combat-related actions.
+
+To have a functional action, you must supply your subclass of Action with some or all of the at_creation, at_attempt, at_completion, at_failure and at_cancel  methods. These will overload the methods in the Action class. at_creation, at_completion, at_failure and at_cancel should always call [super](https://docs.python.org/2/library/functions.html#super), unless you intend to re-implement some of the base functionality of the action system itself. at_attempt simply returns True by default, meaning that the action always succeeds, and thus at_attempt can be safely overriden.
+
+To create an action and initialize it, simply create an object of your subclass of Action. Say you have a class MyAction that inherits from Action; then simply do MyAction(owner, room, target, etc). As long as the original at_creation method of the Action class is called (whether directly or through super), your action will be properly processed so that it will show up either in the list of ongoing actions in the room or in the owner's actions queue.
+
+
+## Commands
+
+There are two command sets in the evennia.contrib.actions module, ActionCmdSet and ActionDebugCmdSet. The commands in ActionCmdSet are meant for all characters and include the following:
+
+#### alarm
```

Comment:
Maybe I'm not familiar with the original intent of this system, but "alarm" seems like a strange term for going into turn-based mode. As a new user I would personally have more expected the command for switching to this mode to be named maybe "turns", "pause", "turnbased" or something to that effect. Is there a particular motivation for chosing the term "alarm"?